nixos/modules/services/web-servers/nginx/location-options.nix

```diff
@@ -71,6 +71,16 @@ with lib;
         These lines go to the end of the location verbatim.
       '';
     };
+
+    priority = mkOption {
```

Comment:
How is this option used?

Reply:
```
            locations = {
              "= /robots.txt" = {
                priority = 100;
                extraConfig = ''
                  allow all;
                  log_not_found off;
                  access_log off;
                '';
              };
...
```

Implementation in `nixos/modules/services/nginx/default.nix`:
```
...
  '') (sortProperties (mapAttrsToList (k: v: v // { location = k; }) locations)));
...
```